napari/components/dims.py

```diff
@@ -347,6 +347,12 @@ def reset(self):
         self.current_step = (0,) * self.ndim
         self.order = tuple(range(self.ndim))
 
+    def transpose(self):
+        """Transpose displayed dimensions."""
```

Comment:
Just a small clarification of the behavior in 3D mode.

```suggestion
        """Transpose displayed dimensions.
        
        This swaps the order of the last two displayed dimensions.
        The order of the displayed is taken from Dims.order.
        """
```